codec-mqtt/src/test/java/io/netty/handler/codec/mqtt/MqttCodecTest.java

```diff
@@ -297,6 +302,126 @@ public void testUnknownMessageType() throws Exception {
         }
     }
 
+    @Test
+    public void testConnectMessageForMqtt31TooLarge() throws Exception {
+        final MqttConnectMessage message = createConnectMessage(MqttVersion.MQTT_3_1);
+        ByteBuf byteBuf = MqttEncoder.doEncode(ALLOCATOR, message);
```

Comment:
I think you want to call `byteBuf.release()` once done in the test

Reply:
You are right. It slipped my mind.